Task20_TC/Task20_TC/LoginTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using System;
+
+namespace Task20_TC
+{
+    [TestFixture]
+    class LoginTests
+    {
+        private const string USEREMAIL = "seleniumtests@tut.by";
+        private const string USERPASSWORD = "123456789zxcvbn";
+        
+        private ChromeDriver chromeDriver;
+
+        [SetUp]
+        public void SetUp()
+        {
+            chromeDriver = new ChromeDriver();
+            chromeDriver.Manage().Window.Maximize();
+            chromeDriver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(50);
+            chromeDriver.Url = "https://www.tut.by/";
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            chromeDriver.Quit();
+        }
+
+        [Test]
+        [TestCase("Selenium Test")]
+        public void CorrectUserLogin(string expectedResult)
```

Comment:
You don't need pass parameters for this test. Just create local variable expectedResult with needed value.